docs/developer-guide/mapstore-migration-guide.md

```diff
@@ -20,6 +20,53 @@ This is a list of things to check if you want to update from a previous version
 - Optionally check also accessory files like `.eslinrc`, if you want to keep aligned with lint standards.
 - Follow the instructions below, in order, from your version to the one you want to update to.
 
+## Migration from 2024.01.00 to 2024.01.02
+
+### Option to hide the group info of logged in user from user details modal window
+
+Recently, we have added the option to hide the `user group info` from the user details modal.
+To enable this, you have to add a cfg in all `Login` plugin into `localConfig.json` like:
+
+```json
+{
+    "name": "Login",
+    "cfg": { "toolsCfg": [{"hideGroupUserInfo": true}] }
+}
+```
+
+where the first index of toolsCfg is for `userDetails` component that is responsible for displaying the user details including `user group info`
+
+!!! note important notes should be considered:
+
+- if you have customized the Login plugin and in particular the order of toolsCfg, make sure to override the correct one as the propagation of cfg for the tools is based on index value.
+
+### Integration with openID Connect
+
+A generic OpenID Connect (OIDC) authentication support has been introduced in MapStore. This feature allows to authenticate users using an OIDC provider, like Keycloak, Okta, Google, Azure, etc.
+
+To provide this functionality, it is necessary to update the project's `geostore-spring-security.xml` file, if the default one is not used.
+If you are using the default one, you can skip this step.
+
+Here the changes to apply if needed:
+
+```diff
```

Comment:
@mahmoudadel54 please fix conflicts and double check your PR after you create it comparing it with the [previous one](https://github.com/geosolutions-it/MapStore2/pull/10415/files#diff-3599ba7c628c7c764665046828bad74c0c8576aad03f5497cf426b59010a6d07R23) in case of backports to ensure you are contributing the same contents.